CONTRIBUTING.md

```diff
@@ -15,10 +15,15 @@ For installation from GitHub, call:
 ```bash
 git clone --recursive https://github.com/laminlabs/lamindb
 pip install laminci
+uv venv
```

Comment:
I would prefer to keep the instructions more agnostic and not require/suggest `uv`. I am of course a fan and user but if we stick to pip and `python -m venv`, I would be happier. Non `uv` workflows still have significant user bases. `uv` users know when to add the `uv` prefix :)

Reply:
Agreed :)